### PR TITLE
Array type support

### PR DESCRIPTION
**Pull request by vikasgupta78**

### CODE REVIEW

File: common/core/src/main/java/zingg/common/core/executor/Matcher.java [hunk 1]
```diff
@@ -85,6 +85,7 @@ public void execute() throws ZinggClientException {
         try {
 			// read input, filter, remove self joins
 			ZFrame<D,R,C>  testDataOriginal = getTestData();
+			LOG.warn("testDataOriginal schema is " +testDataOriginal.showSchema());
```
Comment (sonalgoyal) [inline common/core/src/main/java/zingg/common/core/executor/Matcher.java:88]: change to debug

File: common/core/src/main/java/zingg/common/core/executor/Trainer.java [hunk 2]
```diff
@@ -38,6 +38,7 @@ public void execute() throws ZinggClientException {
 
 				
 			ZFrame<D,R,C> testDataOriginal = getPipeUtil().read(true, args.getNumPartitions(), false, args.getData());
+			LOG.warn("testDataOriginal schema is " +testDataOriginal.showSchema());
```
[the remark this answers is not in the document]
Comment (vikasgupta78) [reply common/core/src/main/java/zingg/common/core/executor/Trainer.java]: Done in commit c5771006

File: common/core/src/main/java/zingg/common/core/executor/Matcher.java (hunk 1 above)
Comment (vikasgupta78) [reply common/core/src/main/java/zingg/common/core/executor/Matcher.java]: Done in commit c5771006

**Comment by vikasgupta78** [comment]
not merging for now

**Pull request closed without merging**